### ISSUE TITLE
IE9 - skrollr doesn't work and creates long page until screen is repainted?

### ISSUE THREAD

**Issue by tokyobarbs** [Prinzhorn/skrollr#663]
Hey everyone,

I have a bit of a strange issue in IE9.

I have some skrollr transitions which are near the top of my page, and they work fine. However, when skrollr kicks in, it is creating a super long page! There is a ton of black empty space at the bottom of my page and none of the subsequent animations (which are similar to my first one, just a simple transition on left position and opacity) are working. The elements don't show up at all.

Then when I open the web developer tool bar and close again, the issue goes away and the black space at the bottom of the page disappears. The skrollr animations work as expected. So this means when the screen is repainted?

I've validated my html, all good there, and when i disable skrollr, the long page issue isn't there....anyone experienced anything like this before? Any hint at what the problem could be? It's hard to debug, I can't really see what's happening in IE.

Any help appreciated!

Another interesting thing, it works fine in IE8.....stumped.

Thanks,
Craig

**Comment by enriqg9** [comment]
Try setting forceHeight to false:

    skrollr.init({
		forceHeight: false
	});

**Comment by tokyobarbs** [comment]
Hey!

Thanks for the reply. I actually managed to track down the issue. Sorry! I should have closed the issue, but got completely sidetracked. I'll close it now.

It was flexslider (responsive slideshow plugin). Before my flexslider code was kicking in, the 'slides' I had were still momentarily stacked vertically, thus forcing the page to be longer than they should have been and skrollr making the page longer.

To get around it, i had to force a height on my flexslider containers to prevent the page height rendering too high. Fixed the issue nicely!

:)
